drivers/hmis_csv_importer/app/controllers/hmis_csv_importer/import_overrides_controller.rb

```diff
@@ -16,7 +16,8 @@ def index
   end
 
   def create
-    @override = import_override_source.create!(permitted_params.merge(data_source_id: @data_source.id))
+    @override = import_override_source.create(permitted_params.merge(data_source_id: @data_source.id))
+    flash[:alert] = @override.errors.full_messages.join(', ') unless @override.valid?
```

Comment:
This is fine, but I _thought_ `respond_with` usually handled the errors natively.  Maybe we're not using `simple_form_for`?

Reply:
It is using `simple_forms_for`, and  I remember fighting with this one for a while. It looks like it should be showing the errors at the field, but I wasn't able to determine where along the line the form was going against that. Happy to dig more into it to see if anything pops out this time.

Reply:
That's fine, this should get very little use, let's push it over the finish line and see what turns up in QA.